### DISCUSSION TITLE
fabric-triage team

### DISCUSSION THREAD

**Discussion by denyeart** [hyperledger/fabric-protos#221] [General]
We are going to do a trial with a fabric-triage team starting in the fabric-protos repository. The team will consist of non-maintainer contributors who are interested in helping with issue triage (close, convert to discussion, etc) and pull request reviews. The github 'triage' role has more privilege than 'read' but less than 'write' and 'maintain'. This is especially important now that the Hyperledger org and teams have become more locked down.

See details of roles at
https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/managing-repository-roles/repository-roles-for-an-organization

@jt-nti As part of the trial you should have received an invitation as an initial member. Please give it a try and let us know if you are able to 'triage', e.g. see if you have a button to close issues now. If it looks promising, we can apply the team across other fabric repositories and add more members.

**Comment by jt-nti** [comment]
Hi @denyeart, I've accepted the invite, thanks, and I can see a button to close issues on the main fabric repo but not on things like fabric-protos, fabric-gateway, etc.

**Comment by denyeart** [comment]
@jt-nti Thanks. Of course I forgot to add fabric-triage to fabric-protos, but I'm glad it worked in fabric repository. I've now added fabric-triage to every fabric related repository. I plan to add maintainers and other interested contributors with a track record of helping on issues to the fabric-triage team so that they can help to manage issues across any fabric repository, get assigned issues, etc.
